apollo-router/src/response.rs

```diff
@@ -42,12 +42,16 @@ pub struct Response {
 
     #[serde(skip_serializing)]
     pub subselection: Option<String>,
+
+    #[serde(skip_serializing_if = "Vec::is_empty", default)]
+    pub incremental: Vec<IncrementalResponse>,
 }
 
 #[buildstructor::buildstructor]
 impl Response {
     /// Constructor
     #[builder(visibility = "pub")]
+    #[allow(clippy::too_many_arguments)]
```

Comment:
I  think this is done automagically since [0.5.0](https://github.com/BrynCooke/buildstructor/commit/ed8c7e5afa9d4f8bd3269db3625f45c9049355a3)

Reply:
here the lint was still complaining, unfortunately :/

Reply:
interesting,  we should open an issue